### PR TITLE
CATROID-852 Rename General Properties to Look Properties

### PR DESCRIPTION
**Pull request by CodeChamp-SS**
Renamed General Properties to Look Properties in the Properties list in the formula editor.

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by Walcho1125 — APPROVED** [review]
Thank you, well done!

File: catroid/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1899,7 +1899,7 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_functions_lists">Lists</string>
     <string name="formula_editor_logic_boolean">Logical (Boolean) operators and constants</string>
     <string name="formula_editor_logic_comparison">Comparison operators</string>
-    <string name="formula_editor_object_general">General properties</string>
+    <string name="formula_editor_object_general">Look properties</string>
```
Comment (wslany) [inline catroid/src/main/res/values/strings.xml:1902]:
```suggestion
    <string name="formula_editor_object_look">Look properties</string>
```

**Review by wslany — CHANGES_REQUESTED** [review]
Please also rename the id name.

File: catroid/src/main/res/values/strings.xml (hunk 1 above)
Comment (CodeChamp-SS) [reply catroid/src/main/res/values/strings.xml]: done : )

**Review by wslany — CHANGES_REQUESTED** [review]
Please fix the compile errors.

**Comment by CodeChamp-SS** [comment]
> Please fix the compile errors.

done, please do lemme know if any further changes are required.

**Review by Walcho1125 — APPROVED** [review]
Thank you !

**Review by wslany — APPROVED** [review]

**Comment by wslany** [comment]
Thanks!

**Pull request merged**